src/lib/receiver.js

```diff
@@ -170,13 +176,13 @@ function createReceiver (opts) {
   /**
    * Generate shared secret for Pre-Shared Key (PSK) transport protocol.
    *
-   * @return {Object} Object containing destination address and shared secret
+   * @return {PSKParams}
    */
   function generateSharedSecret () {
     const token = base64url(hmacHelper.getPskToken())
     return {
-      destination_account: getAddress() + '.' + pskReceiverId + token,
-      shared_secret: base64url(hmacHelper.getPskSharedSecret(token))
+      destinationAccount: getAddress() + '.' + pskReceiverId + token,
```

Comment:
For now I don't think this return value constitutes a protocol by itself so I think we should go with camelCase